User/tasks.py

```diff
@@ -20,12 +20,12 @@
 
 def list_users(date):
     list_user_today = []
-    all_users = User.objects.all()
-
-    for user in all_users:
-        date_joined = user.date_joined
-        if datetime.date(date_joined) == date:
-            list_user_today.append(user.username)
+    all_users = User.objects.filter(date_joined__year=date.year,
+                                    date_joined__month=date.month,
+                                    date_joined__day=date.day)
+    for user in all_users.iterator():
```

Comment:
co the lam gon hon:

list_user_today = [user.username for user in all_users.iterator()]

Reply:
Dạ em cũng đã test thử phần này với đọc thêm về shared_task và task. 
Thì em hiểu được là shared_task sẽ sử dụng khi hàm hay thư viện đó được tái sử dụng, còn task thì không cần quan tâm đến tái sử dụng.